schema/vrs-source.yaml

```diff
@@ -240,6 +246,43 @@ definitions:
 
     required: [ "subject", "copies" ]
 
+  Genotype:
+    description: >-
+      A set of trans-phased :ref:`MolecularVariation` members, with associated
+      copy counts, across a specified number of genomic locus `copies`.
+    additionalProperties: false
+    type: object
+    properties:
+      type:
+        type: string
+        const: "Genotype"
+        default: "Genotype"
+        description: >-
+          MUST be "Genotype"
+      members:
+        type: array
+        uniqueItems: true
+        minItems: 1
+        items:
+          $ref: "#/definitions/GenotypeMember"
+        description: >-
+          Each GenotypeMember in `members` describes a :ref:`MolecularVariation`
+          and the count of that variation at the locus.
+      copies:
+        oneOf:
+          - $ref: "#/definitions/Number"
+          - $ref: "#/definitions/IndefiniteRange"
+          - $ref: "#/definitions/DefiniteRange"
+        description: >-
+          The total number of copies of all :ref:`MolecularVariation` at this locus,
```

Comment:
Is this an accurate description of the copies field? Isn't the Genotype.copies really about the "ploidy" of the system? By saying it is the total number of copies of all MolecularVariation at this locus it presumes a couple things: 1) all GenotypeMembers occupy the same locus (which we are not requiring - I think), and 2) the total number of copies of all MolecularVariation might be misunderstood to not mean "the number of instances of the molecules that contain any one of the GenotypeMembers which make each copy of the members in-trans.".  
It seems like we need to be clearer about what's going on here. I know we get (I think), but we need to explain this very clearly in text so as to reduce confusion.


We should probably add Implementation Guidance to say things like "must be greater than or equal to 1" and "the sum of the member's copies should always be less than or equal to the genotype.copies" and "if the genotype.copies is greater than the sum of the member's copies at any given locus then it can be inferred that the state at the non-specified copies is not explicitly represented."

Reply:
@ahwagner we need to add additional implementation guidance and a reference to how these GenotypeMembers are first _Normalized_ ahead of being digested.  For example, if someone creates a Genotype with 2 separate GenotypeMembers that have the same exact Allele, then it should be reduced into a single GenotypeMember with a count that is the sum of the individual member's counts (which could involve combining indefinite ranges, definite ranges and numbers, of course).  

I do think each Variation class should have a section related to **computed digest notes** which calls out any caveats to be resolved in _normalizing_ the representation before it is digested.  I think as the model gets more and more complex these _normalization_ notes will be critical for generating matching digests for situations that the schema validation cannot catch but are still truly equivalent.

Reply:
we will update the implementation notes to make sure duplicate variations in the .members attributer are not allowed.